apps/rush-lib/typings/jest/jest.d.ts

```diff
@@ -0,0 +1,3 @@
+declare namespace jest {
+  function requireActual(moduleName: string): any;
+}
```

Comment:
Is this not included in the `@types/jest` package?

Reply:
No, I just that it was strange.

---
In reply to: [254808771](https://github.com/Microsoft/web-build-tools/pull/1062#discussion_r254808771) [](ancestors = 254808771)